model_evaluation/jigsaw_evaluation_pipeline.ipynb

```diff
@@ -730,12 +753,12 @@
    "source": [
     "for model_family in MODEL_FAMILIES:\n",
     "  auc_list = []\n",
-    "  for model in model_family:\n",
+    "  for _model in model_family:\n",
     "    fpr, tpr, thresholds = metrics.roc_curve(\n",
     "        test_performance_df['label'],\n",
-    "        test_performance_df[model])\n",
+    "        test_performance_df[_model])\n",
```

Comment:
Why is this being called _model?

Reply:
I changed the variable name of the for loop from model to _model, so that it does not overwrite the global variable 'model'